.github/workflows/code-freeze.yml

```diff
@@ -0,0 +1,53 @@
+name: Code freeze
```

Comment:
1.  It is just for simplicity, once user provides a non-semantic version like `1.23`, it is challenging to go to find latest on 1.23 use that as a base and continue. Once you provide direct version like 1.23.0, I know that I should derive it from 1.23.0. Let say that, we introduced several fix release and version is 1.23.7. Then to introduce a fix release, it is ok to say, create release from 1.23.7. 
2. `v` prefix also breaks some logic in the calculation, and it complaints on several places. For example, since we use v on chart version, it complains about it is not semantically versioned in the validation. (This does not breaks the flow). If the prefix `v` is somehow must have, maybe we can put it in the release name, but for tags, I wouldn't use it since it is not semantic.